draft-xyz-rats-psa-endorsements.md

```diff
@@ -104,24 +111,44 @@ There are three basic types of PSA endorsements:
 * Certification Claims ({{sec-certificates}}), i.e., metadata that describe
   the certification status associated with a PSA device.
 
+Each type is encoded using a separate CoMID.
```

Comment:
Is it mandatory to state that:
Technically an Endorser can stash Reference Values, Identity Claims and Endorsement claims in one CoMID as well ? Correct me, if I am wrong :)

Reply:
We said identity claims and ref-vals MUST go in different CoMIDs because they will be carried over different APIs.

As per cert claims, they could go in the same CoMID, potentially.  But this would make using explicit linking much more problematic.